permissions/plugin-vrealize-automation-8.yml

```
@@ -0,0 +1,7 @@
+---
+name: "vrealize-automation-8-plugin"
+github: "jenkinsci/vrealize-automation-8-plugin"
+paths:
+- "org/jenkins-ci/plugins/vrealize-automation-8"
```

```suggestion
- "io/jenkins/plugins/vrealize-automation-8"
```

See https://github.com/prydin/vrealize-automation-plugin-for-jenkins/blob/master/pom.xml#L11